pymess/backend/dialer/daktela.py

```diff
@@ -139,15 +144,18 @@ def publish_message(self, message):
                 'daktela_statuses': resp_json['result']['statuses'],
             })
 
-            self.update_message(
-                message,
-                state=DialerMessage.STATE.READY,
-                error=resp_json['error'] if len(resp_json['error']) else None,
-                sent_at=tz.now(),
-                extra_data=message.extra_data,
-            )
+            error_message = resp_json.get('error')
+            if error_message:
+                self._update_message_with_error(message, error_message)
```

Comment:
the change is to make it a is_final_state=True because there's no cron job to send error messages.

Reply:
na to je task aby se toto udělalo.

Reply:
btw tady jsi mohl raisnout chybu

Reply:
that would make sense too but I noticed there's this comment at the end of the function.
```
            # Do not re-raise caught exception. We do not know exact exception to catch so we catch them all
            # and log them into database. Re-raise exception causes transaction rollback (lost of information about
            # exception).
```

Reply:
So which error should it raise then?

Reply:
myslel jsem to tak, že když tu raisneš chybu tak to ten except odchytí a zavolá _update_message_with_error. Můžeš raisnout DialerSendingError

Reply:
the formatting of error in JSON is complicated so I left it as it is. It would be ugly otherwise: `raise DialerSendingError(', '.join(error_message) if isinstance(error_message, Sequence) else str(error_message))`